tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -193,6 +224,18 @@ pub struct CompactCiphertextList {
     pub pbs_order: PBSOrder,
 }
 
+impl ParameterSetConformant for CompactCiphertextList {
+    type ParameterSet = CiphertextListConformanceParams;
+
+    fn is_conformant(&self, params: &CiphertextListConformanceParams) -> bool {
+        self.ct_list.is_conformant(&params.ct_list_params)
+            && params.message_modulus == self.message_modulus
+            && params.carry_modulus == self.carry_modulus
+            && params.pbs_order == self.pbs_order
+            && self.degree.0 + 1 == self.message_modulus.0
```

Comment:
choose a convention for degree and message modulus please 🙂 

I would use

self.degree.0 == params.message_modulus.0 - 1

Reply:
It panicked on a test with overflow checks

Reply:
🤔 how ? message_modulus has to be 1 or higher 😵‍💫

Reply:
otherwise it makes no sense right ?

Reply:
Sure, but we can not assume that the degree is correct here

Reply:
Anyway, the expected degree is now taken from the params